patch_auditwheel_whitelist.py

```diff
@@ -15,6 +15,7 @@ def add_zlib_versions():
                 updated_line = line[:-2] + ', "1.2.9", "1.2.12"]'
             else:
                 updated_line = line[:-2] + '"1.2.9", "1.2.12"]'
```

Comment:
Try to add `1.2.13` version for Zlib also, because it has been changed in a docker to `1.2.13`. It may solve an issue with Zlib in tests.

I don't have an opportunity to test it on Linux and with docker now, but as I remember a whitelist was different from time to time and maybe some versions should be added also (I hope not).